easybuild/easyconfigs/l/libobjcryst/libobjcryst-2017.2.2-intel-2020a.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'SCons'
+
+name = 'libobjcryst'
+version = '2017.2.2'
+
+homepage = 'https://github.com/diffpy/libobjcryst'
+description = "ObjCryst++ is Object-Oriented Crystallographic Library for C++"
+
+toolchain = {'name': 'intel', 'version': '2020a'}
+
+source_urls = ['https://github.com/diffpy/libobjcryst/releases/download/v%(version)s/']
+sources = ['libobjcryst-%(version)s.tar.gz']
+patches = [('./libobjcryst_custom_sconscript_local.patch', 1)]
+checksums = ['ad57f4667f6d0b8f0de67a01ecda8e4e09fd0317bf2e0866a4a987b48201583c']
```

Comment:
Also add a SHA256 checksum for the patch file please (`eb --inject-checksums` is your friend here)